baipw/tests/test_utils.py

```diff
@@ -43,39 +43,3 @@ def test_authorise_with_invalid_header(self):
         self.assertEqual(
             str(e.exception), "Invalid format of the authorization header."
         )
-
-
-class TestGetClientIP(TestCase):
-    def setUp(self):
-        self.request = RequestFactory().get("/")
-
-    def test_get_client_ip_from_remote_addr(self):
-        self.request.META["REMOTE_ADDR"] = "192.168.0.17"
-        self.assertNotIn("HTTP_X_FORWARDED_FOR", self.request.META)
-        self.assertIn("REMOTE_ADDR", self.request.META)
-        self.assertEqual(get_client_ip(self.request), "192.168.0.17")
```

Comment:
**Question:** Should we not keep the test for `REMOTE_ADDR`?

Reply:
There are existing tests which cover that `REMOTE_ADDR` is used.